src/types.ts

```diff
@@ -111,6 +111,9 @@ type TWallet = {
       fromChainId?: number
     }
   ) => Promise<string /* | false */> // todo: sendTx reject => false
+  signMessage: (message: string, options?: {
+    fromChainId?: number
+  }) => Promise<string>
```

Comment:
How about flatter? Seems like there won't be any more options.
```suggestion
  signMessage: (message: string, forChainId?: number) => Promise<string>
```

Reply:
same options as `sendTx`. we don't know how this will expand in the future